tests/pytest_loganalyzer/pytest_loganalyzer.py

```diff
@@ -0,0 +1,181 @@
+import sys

```

Comment:
Can avoid prefix 'pytest_'?

Reply:
If add symbolic link to the existed loganalyzer.py with different name then loganalyzer.py, this duplication can be removed.

Reply:
Fixed, added symbolic link.